gen_cert.py

```diff
@@ -652,3 +678,28 @@ def _generate_certificate(
             )
 
         return (download_uuid, verify_uuid, download_url)
+
+    @classmethod
+    def reuse_cert(cls, last_cert, new_course, new_designation):
```

Comment:
I too think the change is fine. 

I could do without the irrelevant build up.  An attribute of static methods is that they don't rely on an instance.  It's hardly the point of them.  Even if it was correct, why are we talking about the point of static methods?    The proposed changes speaks for itself.